### ISSUE TITLE
Preserve file name.

### ISSUE THREAD

**Issue by genemars** [paulcpederson/static-site#11]
For example if the input file is "about.html" currently it will generate:
```
about/index.html
```
Can the output name be changed/preserved using a helper or some option?

**Comment by genemars** [comment]
Ok, I just saw `prettyUrl` option for front matter. But I need this to be the default, a global switch for all files.
Is there such option already?

**Comment by paulcpederson** [comment]
There is no option to globally set this, but I think you could do it with a helper. Something like:

```
module.exports = function (site, cb) {
  site = site.map(page => {
    page.prettyUrl = false
    return page
  })
  cb(null, site)
}
```

Then point to it when running static site:

```
static-site -h path/to/your/helper.js
```

**Comment by genemars** [comment]
I've tried both with `page.data.prettyUrl = false` and `page.prettyUrl = false` but none is working.
The problem is this line:
https://github.com/paulcpederson/static-site/blob/master/lib/frontmatter.js#L12
which expect the flag to be read from the front matter data only.

**Comment by paulcpederson** [comment]
gotcha, can probably change this to check for the definition of the property as well here. Will look into it.

**Comment by genemars** [comment]
Hi paul just wondering if any progress on this already?

**Comment by paulcpederson** [comment]
No haven't had a chance, yet.

**Comment by genemars** [comment]
Ok, no hurry. I'd like to let you know about `zUIx`:
http://genielabs.github.io/zuix/
The `zuix-web-starter` project is powered by `static-site` :)
I'm using static-site and a custom helper for generating component-based websites.

**Comment by paulcpederson** [comment]
OK, I didn't end up allowing this from a helper, but you can now set `prettyUrl: false` in the options object, or use `--prettyUrl false` from the command line to turn them off for an entire site. 

Fixed in `v1.3.4`
